### ISSUE TITLE
Protractor with cucumber AfterStep hook throws TypeError: cucumber_2.AfterStep is not a function

### ISSUE THREAD

**Issue by kalebhushan** [cucumber/cucumber-js#1473]
I am trying to run a cucumber feature file and I would like to capture a screenshot in the cucumber html report after each step of the test. I used a AfterStep hook as follows (I have also used After, Before and BeforeAll hooks all of which run fine):

```
import { AfterStep } from "cucumber";

AfterStep(async function() {
  const screenshot = await browser.takeScreenshot();
  this.attach(screenshot, "image/png"); 
});
```
When I try to run my test, I get an error

```
E/launcher - Error: TypeError: cucumber_2.AfterStep is not a function
    at Object.<anonymous> (SmokeTest\hooks\ScenarioHook.ts:64:1)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at SmokeTest\node_modules\cucumber\lib\cli\index.js:142:42
    at Array.forEach (<anonymous>)
    at Cli.getSupportCodeLibrary (SmokeTest\node_modules\cucumber\lib\cli\index.js:142:22)
    at SmokeTest\node_modules\cucumber\lib\cli\index.js:169:41
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (SmokeTest\node_modules\cucumber\lib\cli\index.js:44:103)
    at _next (SmokeTest\node_modules\cucumber\lib\cli\index.js:46:194)
```
I tried to modify the `node_modules/@types/cucumber/index.d.ts` file and added the following there:

```
export interface Hooks {
   AfterStep(code: HookCode): void;
}
```
Also modified the `node_modules/cucumber/dist/cucumber.js `with the following entry:

```
var AfterStep = methods.AfterStep;
exports.AfterStep = AfterStep;
...
function () {
    ...
    AfterStep: this.defineTestRunHook('afterTestRunHookDefinitions'),
    ...
}
```
However, I still face this error. I didn't have to do all these modifications while using BeforeAll, Before, After hooks. PS: cucumber version used 5.1.0, protractor 5.4.2, typescript 3.6.2

**Comment by davidjgoss** [comment]
Hi @kalebhushan - we just merged support for `BeforeStep` and `AfterStep` via #1416 in the last couple of days, and it's not released yet but should be very shortly with 7.0.0.

**Comment by kalebhushan** [comment]
Perfect. Thank you so much.
